extras/bindings/c/SolverInterfaceC.cpp

```diff
@@ -15,149 +15,150 @@ void precicec_createSolverInterface(
 {
   std::string stringAccessorName(participantName);
   std::string stringConfigFileName(configFileName);
-  impl = new precice::impl::SolverInterfaceImpl(stringAccessorName,
-                                                stringConfigFileName,
-                                                solverProcessIndex, solverProcessSize);
+  interface = new precice::SolverInterface(stringAccessorName,
+                                        stringConfigFileName,
+                                        solverProcessIndex,
+                                        solverProcessSize);
```

Comment:
Can you run format-all on the code-base? This looks odd.

Reply:
Done, thanks!